arangod/VocBase/vocbase.cpp

```diff
@@ -1772,6 +1797,56 @@ void TRI_vocbase_t::addReplicationApplier() {
   _replicationApplier.reset(applier);
 }
 
+arangodb::Result TRI_vocbase_t::toVelocyPack(VPackBuilder& result) const {
+  if (ServerState::instance()->isCoordinator()) {
```

Comment:
Why do we need to ask the agency here? An active `TRI_vocbase_t` object should know its own name and settings...?

Reply:
I am not sure why this code was written, but I did not remove the `if` because I was afraid to break the cluster. Should this be removed in another PR?

Reply:
I still think asking the agency doesn't make sense here if the local vocbase objects have all we need.
The question is, do they have all we need? Can you check?